### PR TITLE
live-common bump (+ flowtype fixes)

### PR DESCRIPTION
**Pull request by valpinkman**
Update live-common with latest things needed for 🌳 & others

in term of QA, you can check:
- https://github.com/LedgerHQ/ledger-live-desktop/pull/2754 passes
- that most parts of the app still works

### CODE REVIEW

**Comment by valpinkman** [comment]
/!\ This update also bumps live-common to 12.5.0

File: src/components/BottomModal.js [hunk 1]
```diff
@@ -50,14 +51,14 @@ const BottomModal = ({
...
+        {...rest}
+        {...backDropProps}
         isVisible={isOpened}
         deviceWidth={width}
         deviceHeight={height}
         useNativeDriver
         hideModalContentWhileAnimating
         style={styles.root}
-        {...backDropProps}
-        {...rest}
```
Comment (gre) [inline src/components/BottomModal.js]: did you check we never need to override the props like `style` ?

Comment (valpinkman) [reply src/components/BottomModal.js]: no i did not.... will fix that

File: yarn.lock [hunk 2]
```diff
@@ -1012,9 +1012,9 @@
     uuid "^3.4.0"
 
 "@ledgerhq/react-native-ledger-core@^4.7.1":
-  version "4.7.1"
-  resolved "https://registry.yarnpkg.com/@ledgerhq/react-native-ledger-core/-/react-native-ledger-core-4.7.1.tgz#47de6a004b2437fa1067270fd475ec08f266b5bc"
-  integrity sha512-1cZdFNlb5qqD+uXQnn64R+1qAGpe7n72+p0YVFp3m/TNmgxWrU3KPCwVKPruCnJhndbdYNIM1YdNcg4K5elBbg==
+  version "4.9.1"
```
Comment (gre) [inline yarn.lock]: mmh i'm not sure if this was intended 🤔

**Comment by gre** [comment]
@valpinkman conflict 😭

**Pull request merged**